### ISSUE TITLE
Will it be compatible with fugu15(Dopamine)

### ISSUE THREAD

**Issue by LeeSin-Su** [frida/frida#2498]
I got a problem on 16.0.18 version:
 ”Failed to enumerate applications:this feature requires an iOS Developer Disk Image to be mounted; run Xcode briefly or use ideviceimagemouter to mount one manually“
 iOS 15.1.1 13PM with fugu15 jailbreaken

**Comment by miticollo** [comment]
Did you install the DEB of `frida-server` on Sileo/Filza?

**Comment by LeeSin-Su** [comment]
> Did you install the DEB of `frida-server` on Sileo/Filza?

Yes

**Comment by miticollo** [comment]
Okay. Open a terminal or a SSH session and run `frida-server` if everything is correct you will get an error message that says that you can't bind the the port because is already busy.

**Comment by LeeSin-Su** [comment]
> Okay. Open a terminal or a SSH session and run `frida-server` if everything is correct you will get an error message that says that you can't bind the the port because is already busy.

Yes, I run frida-server in NewTerm and feedback "Unable to start: Error binding to address 127.0.0.1:27042: Address already in use"

**Comment by miticollo** [comment]
If you use Dopamine everything seems correct.

**Comment by miticollo** [comment]
You can't use root login for a security reason. But you can `ssh mobile@<ip>` then `sudo su` password is `alpine`. Remember also `su root` doesn't work but mobile is a sudoer user.

**Comment by LeeSin-Su** [comment]
> You can't use root login for a security reason. But you can `ssh mobile@<ip>` then `sudo su` password is `alpine`. Remember also `su root` doesn't work but mobile is a sudoer user.

Thanks, I try it

**Comment by LeeSin-Su** [comment]
> > You can't use root login for a security reason. But you can `ssh mobile@<ip>` then `sudo su` password is `alpine`. Remember also `su root` doesn't work but mobile is a sudoer user.
> 
> Thanks, I try it

It work thanks for you help. But I have another question：
I connect iDevice to PC using USB lightning cable，and I run frida-ps -Ua in cmd, there always feedback No running applications.

**Comment by miticollo** [comment]
Yeah I know the [issue](https://github.com/frida/frida/issues/2288#issuecomment-1537228187)😭.
Anyway I hope that you can use it even if this disadvantage.

**Comment by LeeSin-Su** [comment]
> Yeah I know the [issue](https://github.com/frida/frida/issues/2288#issuecomment-1537228187)😭. Anyway I hope that you can use it even if this disadvantage.
> 
> And try `frida-ps -U`, it works!

“frida-ps -U” is work

**Comment by ch0mie** [comment]
you need to  build a monkeydev project  on xcode, make your iphone connect the mac, and select your connected device, end,build the project。after then, frida client connects again, it will normally work.
![image](https://github.com/frida/frida/assets/109508641/35dde1b5-d0d3-4879-90d8-dbe7a38a1c76)
